premis3.owl

```diff
@@ -0,0 +1,733 @@
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/environmentFunctionType"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/InstitutionalPolicy">
+    <rdfs:label>Institutional Policy</rdfs:label>
+    <rdfs:comment>A policy decision made by an organization.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/RightsBasis"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/License">
+    <rdfs:label>License</rdfs:label>
+    <rdfs:comment>A license agreement or other legal document that grants rights.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/RightsBasis"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Object">
+    <rdfs:label>Object</rdfs:label>
+    <rdfs:comment>Discrete unit of information subject to digital preservation. Subclasses of Object are
+      Intellectual Entity, Representation, File and Bitstream.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.w3.org/ns/prov#Entity"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Organization">
+    <rdfs:label>Organization</rdfs:label>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/Agent"/>
+    <rdfs:subClassOf rdf:resource="http://xmlns.com/foaf/0.1/Organization"/>
+    <rdfs:subClassOf rdf:resource="http://www.w3.org/ns/prov#Organization"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Person">
+    <rdfs:label>Person</rdfs:label>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/Agent"/>
+    <rdfs:subClassOf rdf:resource="http://www.w3.org/ns/prov#Person"/>
+    <rdfs:subClassOf rdf:resource="http://xmlns.com/foaf/0.1/Person"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/PreservationPolicy">
+    <rdfs:label>PreservationPolicy</rdfs:label>
+    <rdfs:comment>Information indicating the decision or policy on the set of preservation functions to be
+      applied to an object and the context in which the decision or policy was made. Note that in addition to
+      subclasses declared at presLevType, SignificantProperties is also a subclass of PreservationPolicy.
+      Implementers may also wish to create locally-defined subclasses.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://purl.org/dc/terms/Policy"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/preservationLevelRole"/>
```

Comment:
Depending on the decision we'll make, this vocabulary should be suproperties of premis:hasPolicy.